core/scss/components/main-nav/_main-nav.scss

```diff
@@ -30,7 +33,7 @@
 // Style guide: Components.Main Navigation
 //
 
-$custom-active: #ec0513; //custom high contrast active/current link border for accessibility
+$high-contrast-red: #ec0513; //custom high contrast active/current link border for accessibility
```

Comment:
if we are going to add variables to components we should prefix them with the component name so we don't run into global naming collisions.

Reply:
Otherwise, roll this back up to the colors variable file.